### ISSUE TITLE
The loss is not reduced during training phase

### ISSUE THREAD

**Issue by TL9792** [Yebulabula/DisC-Diff#4]
Hello, I hope this message finds you well. 

When reproducing your code, the loss is not reduced. One subject used for training has high-resolution and low-resolution MRIs with three modalities, including T1w MRI, T2w MRI, and FLAIR MRI. 

For example, the xt is noisy HR T1w MR images, the y is the corresponding LR MR image, and the v includes LR T2w MRI and LR FLAIR MRI. 
The figure blow is the training file. Could you please tell me what causes the training loss to either not decrease or decrease only negligibly?

Looking forward to your response! 

<img width="1335" height="978" alt="Image" src="https://github.com/user-attachments/assets/c3609cb5-f565-4a9e-a4bc-a2e03aa7dae1" />>

v
